Dockerfile-Prerelease

```diff
@@ -1,20 +1,22 @@
-FROM alpine:3.17.0
+FROM alpine:3.19.1
```

Comment:
This file is a straight copy from the current `Dockerfile`, and the only change is this alpine bump.